extensions/amp-accordion/amp-accordion.md

```diff
@@ -5,7 +5,7 @@ formats:
   - ads
   - email
 teaser:
-  text: Provides a way for viewers to have a glance at the outline of the content and jump to a section of their choice at will.
+  text: A clickable stacked list of headers that reveal or hide content associated with them.
```

Comment:
Nit: Is 'clickable' right here? Do we use this term for predominantly mobile usage?

Alternative: A stacked list of headers that reveal or hide content associated with them via user-interaction.